pombola/core/models.py

```diff
@@ -579,7 +579,7 @@ def constituency_offices(self):
 
         This is specific to the South African site (ZA).
         """
-        contacts = self.position_set.filter(title__slug="constituency-contact").currently_active()
+        contacts = self.position_set.filter(organisation__kind__slug__in=['constituency-area', 'constituency-office']).currently_active()
```

Comment:
This is to show the constituency office(s) a person is linked to on their profile page even if they're not a "constituency contact."

![2020-04-07-15:38](https://user-images.githubusercontent.com/4767109/78675841-ea2ec900-78e5-11ea-8ff5-9606adddd6b6.png)